api/src/main/java/org/pmiops/workbench/firecloud/FireCloudService.java

```diff
@@ -89,6 +89,15 @@ FirecloudWorkspaceDetails cloneWorkspace(
 
   FirecloudWorkspaceACL getWorkspaceAclAsService(String workspaceNamespace, String firecloudName);
 
+  /**
+   * Make a Terra PATCH request with a list of ACL update requests for a specific workspace. Only
+   * makes the changes specified. Choose the access level "NO ACCESS" to remove access.
+   *
+   * @param workspaceNamespace the Namespace (Terra Billing Project) of the Workspace to modify
+   * @param firecloudName the Terra Name of the Workspace to modify
+   * @param aclUpdates
+   * @return
+   */
   FirecloudWorkspaceACLUpdateResponseList updateWorkspaceACL(
```

Comment:
Important!  Terra already uses patch semantics.  We were doing transformations of lists in our app unnecessarily.